app/models/invenio_rdm_record_converter.rb

```diff
@@ -561,6 +563,7 @@ def original_identifiers(identifiers)
       elsif identifier.include?("ISBN")
         id = identifier.gsub(/\(ISBN.*\)/, "").strip
         scheme = "isbn"
+        isbn_count += 1
```

Comment:
Increment each time an ISBN is found rather than setting this as true which would lead to the bug @fenekku pointed out below. This seems like a simple enough approach, but I don't love it.